src/main/java/dev/unnm3d/rediseconomy/currency/CurrenciesManager.java

```diff
@@ -54,8 +54,8 @@ public CurrenciesManager(RedisManager redisManager, RedisEconomyPlugin plugin, C
         this.configManager = configManager;
         this.currencies = new HashMap<>();
         try {
-            this.nameUniqueIds = loadRedisNameUniqueIds().toCompletableFuture().get(1, TimeUnit.SECONDS);
-            this.lockedAccounts = loadLockedAccounts().toCompletableFuture().get(1, TimeUnit.SECONDS);
+            this.nameUniqueIds = loadRedisNameUniqueIds().toCompletableFuture().get(10, TimeUnit.SECONDS);
```

Comment:
Why 10 seconds. if you have that latency it means your redis is down.
Redis isn't like MySQL, it doesn't take ages to load data.
I'm ok if you change it to 3 or 4 secs